### ISSUE TITLE
Kavita uses spaces to count words in languages that don't use spaces as word delimiters. 

### ISSUE THREAD

**Issue by Hobogrammer** [Kareadita/Kavita#2094]
**Describe the bug**
Kavita calculates book length with words by using spaces as word delimiters which leads Kavita to underestimate the length of books and their estimated reading time in languages that do not use spaces as word delimiters. Instead of being a word count, it ends up being a count of sentences or paragraphs.

https://github.com/Kareadita/Kavita/blob/baf7c9eb926c2c7d9c61b68f113d47c11e9fadd7/API/Services/Tasks/Metadata/WordCountAnalyzerService.cs#L241

```
 private static async Task<int> GetWordCountFromHtml(EpubLocalTextContentFileRef bookFile)
    {
        var doc = new HtmlDocument();
        doc.LoadHtml(await bookFile.ReadContentAsync());

        var textNodes = doc.DocumentNode.SelectNodes("//body//text()[not(parent::script)]");
        if (textNodes == null) return 0;
        return textNodes
            .Select(node => node.InnerText.Split(' ', StringSplitOptions.RemoveEmptyEntries)
                .Where(s => char.IsLetter(s[0])))
            .Sum(words => words.Count());
    }
```

**To Reproduce**
1. Add a book in Japanese/Chinese/Thai or any other language that doesn't use spaces to separate words.
2. Observe the low word and reading estimation time.

**Expected behavior**
In languages that don't use spaces to separate words, such as Japanese, Kavita should uses a character count instead of a word count to indicate book length and calculate estimated reading time.

**Comment by majora2007** [comment]
Please provide an example book that is not copyrighted for me to test against.

**Comment by Hobogrammer** [comment]
[石橋 忍月 - 罪過論.zip](https://github.com/Kareadita/Kavita/files/11853992/Shaku.Kyou.Nin.Getsu.Zai.Ka.Ro.-.Shaku.Kyou.Nin.Getsu.zip)

Converted via calibre from the xhtml files available here: https://www.aozora.gr.jp/cards/000102/card945.html

**Comment by majora2007** [comment]
Okay, this seem doable. But the question arises, on the series detail, should I still show Length as words or characters? My thinking is keeping as Words, as that's an easy concept to grasp, and I will just convert the characters to words.

**Comment by Hobogrammer** [comment]
I think showing the length as characters is the way to go in this case. At least this is how I normally see japanese text measured when page counts aren't available.

**Comment by majora2007** [comment]
This will be released in v0.7.4.
